RoaringBitmap/src/main/java/org/roaringbitmap/BitmapContainer.java

```diff
@@ -1665,6 +1657,15 @@ public BitmapContainer toBitmapContainer() {
     return this;
   }
 
+  @Override
+  public void copyBitmapTo(long[] words, int position) {
+    System.arraycopy(bitmap, 0, words, position, bitmap.length);
+  }
+
+  public void copyBitmapTo(long[] words, int position, int length) {
```

Comment:
I am doubtful that this function is ever needed. I think that bitmap.length and BLOCK_LENGTH should always be equal.

???

Reply:
It's to allow for a truncated array copy if a `BitmapContainer` is the last container.